gcp/tpu_zone_list.go

```diff
@@ -0,0 +1,28 @@
+package gcp
+
+import (
+	"context"
+
+	"github.com/turbot/steampipe-plugin-sdk/v5/plugin"
+)
+
+// List of zones where TPUs are available
+// Source: https://cloud.google.com/tpu/docs/regions-zones
+var tpuSupportedZones = []string{
```

Comment:
Should we hardcode the list of zones?

I noticed that there’s an [API available](https://cloud.google.com/tpu/docs/reference/rest/v2/projects.locations/list) to dynamically list the supported regions for TPUs.

We could follow a similar approach to what’s implemented here: [compute_location_list.go](https://github.com/turbot/steampipe-plugin-gcp/blob/main/gcp/compute_location_list.go)